factcast-store-pgsql/src/test/java/org/factcast/store/pgsql/internal/listen/PgListenerTest.java

```diff
@@ -46,108 +49,154 @@
     @Mock
     AsyncEventBus bus;
 
-    @Mock
+    @Mock(answer = Answers.RETURNS_DEEP_STUBS)
```

Comment:
Mocks did not work?

Reply:
no, RETURN_DEEP_STUBS is exactly what helped: https://www.javadoc.io/doc/org.mockito/mockito-core/2.6.9/org/mockito/Mockito.html#RETURNS_DEEP_STUBS